### ISSUE TITLE
Add option to remove author from the admin

### ISSUE THREAD

**Issue by yakky** [aldryn/aldryn-newsblog#64]
Some deployment does not need any author field, keeping it can be confusing for the users

**Comment by mikek** [comment]
Can be possibly added through `NewsBlogConfigForm`

**Comment by mikek** [comment]
@yakky, @tom I've completely forgot to consider doing this through `NewsBlogConfigForm` in #77 Can we use the simple setting approach now and look into the per instance configuration in some next iteration?

**Comment by yakky** [comment]
Sure, it makes perfectly sense

**Comment by intellectronica** [comment]
Sure, makes sense.

**Comment by yakky** [comment]
@mikek This issue is meant to completyely remove the "author" field in the admin, actually

**Comment by mikek** [comment]
Sorry, just noticed this is another issue. Reopening.

**Comment by mkoistinen** [comment]
What's the consensus on this? Can I suggest we make this setting a boolean on the NewsBlogConfig rather than creating new settings entries. This would also allow a Blog instance of this app to use an author field, whilst allowing a News instance to ignore it in the same project.

**Comment by stefanfoulis** [comment]
+1 on making this a setting of NewsBlogConfig. This kind of thing is one of the main reasons it exists. News entries might not need authors. Blog entries usually do. In any case, author should *not* be a mandatory field.
And it should be auto-populated with the following priorities:
1. the current user already has a Person associated: use it
2. the current user has a first_name and last_name: auto create a Person and associate it.
3. set author to None.

Templates should handle missing authors gracefully.

If NewsBlogConfig says there should be no authors:
* hide the author field in admin

@intellectronica @FinalAngel @yakky what do you think?

**Comment by intellectronica** [comment]
Yes to all except for auto-creating a Person. IMO that's overreaching.

**Comment by yakky** [comment]
@stefanfoulis +1

**Comment by FinalAngel** [comment]
@stefanfoulis +1
